pkgs/development/interpreters/python/buildpypi.nix

```diff
@@ -0,0 +1,35 @@
+/* This function wraps buildPythonPackage and uses pre-fetched data from PyPI to build a package.
+*/
+{ buildPythonPackage
+, fetchurl
+, fetchgit
+, lib
+}:
+
+
+{ name
+, version ? ""
+, ... } @ attrs:
+
+let
+  pypi-src = fetchgit {
```

Comment:
Is fetchgit required here?
The good thing about your github structure is, that it is pretty readable. 
Could we generate a compressed version of your directory structure for downloading automatically via travis-ci.
Maybe putting all packages with the same prefix into the same file and compress the whole thing with `xz` could reduce the overall size.